src/components/tenancy/tenancy-form.tsx

```diff
@@ -306,7 +306,7 @@ export const TenancyForm = ({
                         onChange={(e) => handleBasicInputChange(e)}
                         label={'Rent Account (IBAN)'}
                         helperText={'Provide a valid IBAN'}
-                        inputProps={{ minLength: 34 }}
+                        inputProps={{ minLength: 19 }}
```

Comment:
Die `34` war echt grad etwas voreilig von mir. Hab grad nochmal gegoogelt. In der Schweiz muss die IBAN 21 Zeichen haben: https://wise.com/gb/iban/switzerland
![image](https://user-images.githubusercontent.com/20950007/148948059-82bfda32-53b7-4f11-9d3f-5a060d63cc92.png)

Wie kommst du auf 19?